app/assets/src/components/Header.jsx

```diff
@@ -56,31 +56,34 @@ class Header extends React.Component  {
 
   render() {
     let host_genome_names = this.host_genome_names;
-    let cli_modal = this.demoUser != 1 ? (
-      <div id="cli_modal" className="modal project-popup">
-        <div className="modal-content">
-          <p>1. Install and configure the Amazon Web Services Command Line Interface (AWS CLI).</p>
-          <p>2. Install the IDseq CLI:</p>
-          <p><span className="code">pip install git+https://github.com/chanzuckerberg/idseq-cli.git</span></p>
-          <p>3. Upload a sample using a command of the form:</p>
-          <div className="code center-code">
-            <p>idseq -p '<span className="code-to-edit">Your Project Name</span>' -s '<span className="code-to-edit">Your Sample Name</span>' \</p>
-            <p> -u https://idseq.net -e <span className="code-personal">{this.userDetails.email}</span> -t <span className="code-personal">{this.user_auth_token}</span> \</p>
-            <p> --r1 <span className="code-to-edit">your_sample_R1</span>.fastq.gz --r2 <span className="code-to-edit">your_sample_R2</span>.fastq.gz</p>
+    let cli_modal;
+    if (this.userSignedIn) {
```

Comment:
why not just 
if (this.userSignedIn && this.demoUser !== 1) {
}
you don't need another ternary operator.